matter_server/client/models/device_types.py

```diff
@@ -154,6 +154,30 @@ class DimmableLight(DeviceType, device_type=0x0101):
     }
 
 
+class MountedOnOffControl(DeviceType, device_type=0x010F):
+    """Mounted On/Off Control."""
+
+    clusters = {
```

Comment:
At least for other device types like `DimmableLight` we have the Base Device Type clusters here too (`Descriptor` and `Binding`).

Also, `LevelControl` is an optional cluster, and so is `OccupancySensing`. I think we typically add optional clusters here.